shared/middleware/channels_api.rb

```diff
@@ -181,8 +207,11 @@ class ChannelsApi < Sinatra::Base
 
     dont_cache
     content_type :json
-
-    value = StorageApps.new(storage_id('user')).reset_abuse(id)
+    begin
+      value = StorageApps.new(storage_id('user')).reset_abuse(id)
+    rescue ArgumentError, OpenSSL::Cipher::CipherError
+      bad_request
+    end
```

Comment:
Too bad these are all different calls and can't easily be consolidated into a single begin/rescue block :(